rakelib/plugin.rake

```diff
@@ -61,6 +61,8 @@ namespace "plugin" do
     Dir["logstash-core*.gem"].each do |gem|
       rm(gem)
     end
```

Comment:
I don't think it's worth bothering... but we can open an issue if there is a good reaso to be more specific